docs/Features/Vanity.md

```diff
@@ -0,0 +1,34 @@
+# Vanity
+Satisfy all of your cosplay needs with easy-to-use, non-intrusive vanity system.
+Access its menu by right-clicking any of your equipped items and choosing "Vanity..." option.
+![Vanity submenu](img/vanity/submenu.png)
+
+From here you will have access to a number of tabs packed with customization features.
+![Tabs overview](img/vanity/tabs.png)
+
+## Transmog tab ![Transmog tab icon](img/vanity/transmog_tab_icon.png)
+![Transmog tab showcase](img/vanity/transmog_tab.png)
+Equipment tab lets you seamlessly transmogrify the appearance of your weapons and armors.  
+The item you've used to access this tab is its focus, indicated by the dropdown list at the top. To select another item to tinker with you can use rightclick->Vanity... on it or said dropdown list.  
+![Transmog list](img/vanity/transmog_list.png)
+
+### Basics
+Transmog options consist of vanilla armor variants (tell us if we've missed some!) and options added by cosmetic mods. **Support is included for ["Majora's Project Fashion Sins"](https://steamcommunity.com/sharedfiles/filedetails/?id=1815898410), ["Rendal's NPC Armors"](https://steamcommunity.com/sharedfiles/filedetails/?id=1515827226) and ["Visitors from Cyseal"](https://steamcommunity.com/sharedfiles/filedetails/?id=2828485171) mods. Tell us if you think we should include more!**  
+Using transmog is as easy as clicking on your desired option, no extra items or materials required. **This change is purely cosmetic!**  
```

Comment:
This would be the right place to (explicitly) mention that there are no gameplay side-effects - as that is quite a huge advantage over the prehistoric vanity mods